docs/faq.md

```diff
@@ -88,3 +88,9 @@ influence their behavior. While Black does its best to recognize such comments a
 them in the right place, this detection is not and cannot be perfect. Therefore, you'll
 sometimes have to manually move these comments to the right place after you format your
 codebase with _Black_.
+
+## Can I run black with PyPy?
+
+Yes, there is _experimental_ support for PyPy 3.7 and higher. You cannot format Python 2
```

Comment:
Is there any reason other than Python 2 support to call this experimental?

For what it's worth we're about to deprecate Python 2 support anyway.

Reply:
My main thought on is that while it passes all of the tests and I've used it locally on files, there might still be edge cases where it fails while cpython doesn't.

I don't think we'll discover those edge cases without a lot more people trying it out. But yes, maybe that's not enough to say it's "experimental".

Reply:
OK, I've removed these bits from the faq.